src/libraries/Microsoft.PowerFx.Core/Functions/UserDefinedFunction.cs

```diff
@@ -184,7 +185,11 @@ public UserDefinedFunction WithBinding(INameResolver nameResolver, IBinderGlue b
             return func;
         }
 
-        private static readonly ISet<string> _restrictedUDFNames = new HashSet<string> { "Type", "IsType", "AsType" };
+        private static readonly ISet<string> _restrictedUDFNames = new HashSet<string>
+        {
+            "Type", "IsType", "AsType", "Set", "Collect",
+            "ClearCollect", "UpdateContext", "Navigate", "Select", "Reset",

```

Comment:
https://github.com/microsoft/Power-Fx/blob/bd033f54ee476ea78467eb44ec8e1ff057ff299c/src/libraries/Microsoft.PowerFx.Core/Texl/BuiltinFunctionsCore.cs#L25

dont we already block these names here ?

Reply:
Changed so we would allow UDFs to be named after existing functions besides this restricted list